lua/nui/popup/init.lua

```diff
@@ -304,6 +304,56 @@ function Popup:set_size(size)
   end
 end
 
+function Popup:set_position(position, relative)
+  local props = self.popup_props
+
+  local container_info = get_container_info(self)
+  props.position = calculate_window_position(position, props.size, container_info)
+
+  self.border:reposition()
```

Comment:
This should go after the new `self.win_config` of the Popup is set. Border will need to use these values from Popup.

Reply:
Ok awesome, let me try again with your changes.